java-spiffe-core/src/test/java/io/spiffe/workloadapi/DefaultWorkloadApiClientTest.java

```diff
@@ -238,6 +241,7 @@ void testFetchJwtSvidPassingSpiffeId() {
             assertEquals(SpiffeId.parse("spiffe://example.org/test"), jwtSvid.getSpiffeId());
             assertTrue(jwtSvid.getAudience().contains("aud1"));
             assertEquals(3, jwtSvid.getAudience().size());
+            assertEquals(jwtSvid.getHint(), "external");
```

Comment:
```suggestion
            assertEquals("external", jwtSvid.getHint());
```
nit: ordering of args is `expected, actual`